java/src/processing/mode/java/runner/Runner.java

```diff
@@ -659,6 +670,9 @@ public static boolean handleCommonErrors(final String exceptionClass,
       }
     } else if (exceptionClass.equals("java.lang.UnsatisfiedLinkError")) {
       listener.statusError("A library used by this sketch is not installed properly.");
+      if (PApplet.platform == PConstants.LINUX) {
+        System.out.println(message);
```

Comment:
Related to #158 and #2827: It might be better to use `err.println` to write the output to the window of the sketch and not to the active window.